### PR TITLE
Add Oxocarbon Light theme (& small enhancements to Oxocarbon Dark)

### PR DESCRIPTION
**Pull request by xstyyy**
Hello. This PR adds the Light version of the [Oxocarbon](https://github.com/nyoom-engineering/oxocarbon) theme.
It also adds coloring related to the bracketmatch plugin to the dark as well as light version of the theme.

<details><summary>preview image (flashbang warning)</summary>

</details>

### CODE REVIEW

**Review by Guldoman — COMMENTED** [review]
Could you add a `drag_overlay` color? At the moment it's invisible.

**Comment by xstyyy** [comment]
> Could you add a `drag_overlay` color? At the moment it's invisible.

sure!

File: colors/oxocarbon-dark.lua [hunk 1]
```diff
@@ -56,6 +55,17 @@ style.divider = base00 -- borders
...
+style.drag_overlay = { common.color "rgba(255, 255, 255, 0.30)" }
+style.drag_overlay_tab = base03
+
+style.scrollbar = base01 -- scrollbar button when not interacted
+style.scrollbar2 = base03 -- button when you hover over
+style.scrollbar_track = base01 -- scrollbar track
+
+style.nagbar = base10
+style.nagbar_text = base01
+style.nagbar_dim = { common.color "rgba(0, 0, 0, 0)" }
```
Comment (Guldoman) [inline colors/oxocarbon-dark.lua]: Why fully transparent?

Comment (xstyyy) [reply colors/oxocarbon-dark.lua]: Felt like it looked right, but now that you mentioned it, it is probably a good idea to atleast make it darken the screen a little bit

**Comment by Guldoman** [comment]
Looks good, thank you!

**Pull request merged**